rele/client.py

```diff
@@ -114,7 +114,7 @@ def publish(self, topic, data, blocking=False, **attrs):
         Usage::
 
             publisher = Publisher()
-            future = publisher.publish('topic_name', {'foo': 'bar'}, blocking=True) # noqa
+            future = publisher.publish('topic_name', {'foo': 'bar'}, blocking=True, timeout=10.0) # noqa
```

Comment:
What do you think about updating code usage example with the default `PUBLISHER_TIMEOUT` 3 seconds value?

```suggestion
            future = publisher.publish('topic_name', {'foo': 'bar'}, blocking=True, timeout=3.0) # noqa
```